### PR TITLE
chore: add missing metadata to issues templates

### PR DESCRIPTION
**Pull request by armano2**
without metadata issue templates are not going to work

demo: https://github.com/armano2/typescript-eslint/issues/new/choose

### CODE REVIEW

**Comment by JamesHenry** [comment]
Thanks, I've got a PR coming up for converting typescript-eslint-parser to TS

File: .github/ISSUE_TEMPLATE/typescript-eslint-parser.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+name: Typescript eslint parser
```
Comment (j-f1) [inline .github/ISSUE_TEMPLATE/typescript-eslint-parser.md]: `typescript-eslint-parser`

Comment (armano2) [reply .github/ISSUE_TEMPLATE/typescript-eslint-parser.md]: ok i will update it

**Comment by JamesHenry** [comment]
Thanks, everyone!

**Pull request merged**